doc/architecture/database_schemata.md

```diff
@@ -87,8 +87,62 @@ species }|--|| genus : ""
 genus }|--|| subfamily : ""
 subfamily }|--|| family : ""
 
+maps {
+  INT id PK
+  VARCHAR name "NOT NULL"
+  BOOLEAN is_variant "NOT NULL"
+  VARCHAR version_name
```

Comment:
I was waiting for @NurramoX to finish the layer design so that we could plan the versions table together, since it concerns us both. But I can try to staple the general map table and the individual layers together by myself if you want.